examples/monitoring_user_defined_projects_vars.yaml

```diff
@@ -0,0 +1,7 @@
+---
+
+# User defined project monitoring vars
+enable_user_defined_project_monitoring: false  # Set true to enable role execution
+user_email: ""  # User email to which grant permissions 
+user_defined_app_name: ""  # User defined app name e.g. prometheus-example-app
+user_defined_namespace: ""  # User defined namespace e.g test-namespace
```

Comment:
This is not used anywhere.

Reply:
Used to assign user defined name to the ```set_fact``` variables in the playbook.